.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 npm-debug.log
+.idea/
```

Comment:
Could this go in your `.git/info/exclude` instead, considering the IDE-specific nature of it?